homeassistant/components/netatmo/__init__.py

```diff
@@ -36,16 +74,23 @@
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Required(CONF_SECRET_KEY): cv.string,
         vol.Required(CONF_USERNAME): cv.string,
+        vol.Optional(CONF_WEBHOOKS, default=DEFAULT_WEBHOOKS): cv.boolean,
         vol.Optional(CONF_DISCOVERY, default=DEFAULT_DISCOVERY): cv.boolean,
     })
 }, extra=vol.ALLOW_EXTRA)
 
+SCHEMA_SERVICE_ADDWEBHOOK = vol.Schema({
+    vol.Optional(CONF_URL): cv.string,
+})
+
+SCHEMA_SERVICE_DROPWEBHOOK = vol.Schema({})
+
 
 def setup(hass, config):
     """Set up the Netatmo devices."""
     import pyatmo
 
-    global NETATMO_AUTH
+    global NETATMO_AUTH, NETATMO_WEBHOOK_URL
```

Comment:
We should not use global. If this needs to be accessed from another module, use hass.data to store it.